### ISSUE TITLE
Prerequisites for Xibo CMS 1.7.2 missing php5-zip check

### ISSUE THREAD

**Issue by rittmeier** [xibosignage/xibo#451]
Hello,

I was just installing Xibo CMS 1.7.2 on a SUSE system. The prerequisites check detected a couple of missing librarys correctly. After installing all of them I tried to login and got a blank page. Took some time digging around to find out that the ZIPArchive class of php was missing. On Suse I could install it using the package php5-zip, but it would have been nice if it had been listed in the list of prerequisites. Would be nice if you could add that check for future setups.

Regards,
Florian

**Comment by dasgarner** [comment]
#381 
Should have already been fixed - odd that it isn't!

**Comment by rittmeier** [comment]
Rethinking about that I might have overlooked the item/warning. I now took a look at your source code and saw, that missing that library is considered a warning. As I got a blank page after login it seems that the library is not only used during import / export. Maybe it should be considered a fault if it is missing, so that the user cannot proceed installing Xibo using "Next" without having installed that library before.
